### PR TITLE
Development

### PR DESCRIPTION
**Pull request by suretrust**
Submitting for code review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+</head>
+<body>
+  <header>
+    <i class="nav fas fa-bars left"></i>
+    <i class="nav fas fa-search left"></i>
+    <a href="https://www.nytimes.com/section/science/space" class="nav left">SPACE & COSMOS</a>
+    <a href="index.html">
+      <img class="logo" src="img/The_New_York_Times_logo.png" alt="logo">
```
Comment (zlayabekrija) [inline index.html:17]: image is bigger then anchor, please fix that

File: index.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    <button class="nav right">LOGIN</button>    
+  </header>
+  
+  <div id="main" class="wrap">
+    <article>
+      <h1>Space Ripples Reveal<br />Big Bang’s Smoking Gun</h1>
+
+      <div class="article-img">
+        <img src="https://static01.nyt.com/images/2014/03/18/science/space/18cosmos/18cosmos-superJumbo.jpg?quality=90&auto=webp" alt="">
+        <caption class="img-footer">Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</caption>
```
Comment (zlayabekrija) [inline index.html]: caption should wrap the image, don't put it under the image

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+.icons {
+    float: right;
+}
+
+.icons i {
+    margin-left: 10px;
+}
+
+
+article p {
```
Comment (zlayabekrija) [inline css/style.css:95]: This is big plus for you guys 👍  🥇

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+    clear: both;
+}
+
+.text {
+    margin-left: calc((100% - 600px) / 2);
```
Comment (zlayabekrija) [inline css/style.css]: margin is still to0 big.
remember text-align center :)

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+
+#sidebar ul li img {
+    width: 50px;
+}
+
+
+/***** Footer ****/
+
+footer {
+    clear: both;
```
Comment (zlayabekrija) [inline css/style.css:167]: Great hands up for this.

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Excellent work.
Please add some more text and images.
Also fix the things that are pointed out.
Submit new review request and happy coding :)

File: css/style.css (hunk 4 above)
Comment (jeton-th) [reply css/style.css]: text-align doesn't work in this case, because we want to center the paragraph not the actual text

Comment (jeton-th) [reply css/style.css]: ok we fixed it with:
`margin: 0 auto;`

**Review by zlayabekrija — APPROVED** [review]
Excellent work, thumbs up 👍 from me.
I really liked your comments in css, super descriptive big thumbs up for this.

**Pull request merged**